sse_solver/src/solvers.rs

```diff
@@ -1,7 +1,10 @@
-use ndarray::{Array1, Array2};
-use ndarray_linalg::Norm;
+use core::f64;
```

Comment:
I don't think this import is needed - you can use the one implicitly imported from std::